### DISCUSSION TITLE
React Native 0.65

### DISCUSSION THREAD

**Discussion by softshipper** [expo/expo#14257] [Archived: Questions] [answered]
Hi all 
Are you going to upgrade to React Native 0.65 soon?

Thanks

**Comment by EvanBacon** [comment, marked as answer]
No, we'll be upgrading to react-native 64 in SDK 43, but react-native 65.1 is still buggy and it'll take us a while to ensure it's stable enough to support the entire Expo SDK.

**Comment by EmreAbi** [comment]
Since you guys skipped 0.65, it's already react-native 0.67. any planned upgrade soon?

**Comment by boxtcarpel** [comment]
We're also looking for RN 0.65 (or newer) support. Any updated timeline or expectations on this?
